tardis_kromer_plot.py

```diff
@@ -6,28 +6,121 @@
 import numpy as np
 import astropy.units as units
 import astropy.constants as csts
+import pandas as pd
+
 try:
     import astropy.modeling.blackbody as abb
 except ImportError:  # for astropy version < 2.0
     import astropy.analytic_functions as abb
-
 import matplotlib
 import matplotlib.pyplot as plt
 import matplotlib.patches as patches
 import matplotlib.lines as lines
 import matplotlib.cm as cm
 from tardis_minimal_model import minimal_model
 
+plt.rcdefaults()
+
 logger = logging.getLogger(__name__)
 
-elements = {'neut': 0, 'h': 1, 'he': 2, 'li': 3, 'be': 4, 'b': 5, 'c': 6, 'n':
-            7, 'o': 8, 'f': 9, 'ne': 10, 'na': 11, 'mg': 12, 'al': 13, 'si':
-            14, 'p': 15, 's': 16, 'cl': 17, 'ar': 18, 'k': 19,    'ca': 20,
-            'sc': 21, 'ti': 22, 'v': 23, 'cr': 24, 'mn': 25, 'fe': 26, 'co':
-            27, 'ni': 28, 'cu': 29, 'zn': 30, 'ga': 31, 'ge': 32, 'as': 33,
-            'se': 34, 'br': 35, 'kr': 36, 'rb': 37, 'sr': 38, 'y': 39,  'zr':
-            40, 'nb': 41, 'mo': 42, 'tc': 43, 'ru': 44, 'rh': 45, 'pd': 46,
-            'ag': 47, 'cd': 48}
+"""the elements dictionary looks like this because this is how black linter
+formatted it - until this info is moved and stored in a separate file, it will
+remain here like this"""
+elements = {
```

Comment:
For now, we can put this in a csv file in same directory in this repository, and read it from there - preferably by pandas!

Reply:
I have added this functionality now - see PR #29